notes_app/src/firebase.js

```diff
@@ -0,0 +1,15 @@
+import { initializeApp } from "firebase/app";
+import { getFirestore, collection } from "firebase/firestore"
+
+const firebaseConfig = {
+  apiKey: "AIzaSyCF9CveXrn5ch0lkJhnRrMHoaOPWR65M_Y",
```

Comment:
those are secret keys and shouldn't be pushed to github. 

Please watch https://www.youtube.com/watch?v=FcwfjMebjTU